kong-ngx-build

```diff
@@ -254,6 +283,14 @@ main() {
               || fatal "failed to apply patch: $patch_file"
           done
 
+          if [ $KONG_NGINX_MODULE != 0 ]; then
+            for patch_file in $(ls -1 $DOWNLOAD_CACHE/lua-kong-nginx-module/patches/$OPENRESTY_VER/*.patch); do
+              notice "Applying OpenResty patch $patch_file"
+              patch -p1 < $patch_file \
```

Comment:
I feel very strongly that those patches should belong to openresty-patches, which was meant (and became) to be a repository storing all Kong-needed patches (pending bugfixes, but new features as well). Currently this adds one more repository containing OpenResty patches that users and developers need to track, maintain (e.g. bump version number on OpenResty bumps), etc... 
The only exception is this repository itself, which contains patches that are not Kong-specific (e.g. general bugfixes preventing compilation on some platforms, etc...).
This was also noted in this comment, which also mentions how to make the FFI API safer in case the patch is missing: https://github.com/Kong/lua-kong-nginx-module/pull/6#discussion_r304183779

Reply:
This has been moved in https://github.com/Kong/openresty-patches/pull/46.